src/components/SideBar/AssetPackList/AssetPackList.tsx

```diff
@@ -1,34 +1,26 @@
 import React from 'react'
-import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { T } from 'decentraland-dapps/dist/modules/translation/utils'
 
 import SidebarCard from '../SidebarCard'
 import { Props } from './AssetPackList.types'
 import './AssetPackList.css'
 
-const ethereum = (window as any)['ethereum']
-
 export default class AssetPackList extends React.PureComponent<Props> {
   handlePromoClick = () => {
-    this.props.onOpenModal('AdBlockModal', { origin: 'Item catalog Dapper CTA' })
+    window.open('https://avatars.decentraland.org?utm_source=builder&utm_campaign=catalog')
```

Comment:
Hardcoded URL ⚠️

Reply:
Where do you think it might be a good place to set this constant?

Reply:
Addressed 👍